Sample/Program.cs

```diff
@@ -13,11 +14,41 @@ static async Task Main(string[] args)
             
             await client.GetGroupsAsync();
             await client.GetChatsAsync();
+            ICollection<Contact> contacts = await client.GetContactsAsync();
```

Comment:
We need unit tests or examples or something, but I don't think the getting started sample is the best place to put this. If you want to demo the Contacts API, retrieving them and maybe printing out how many contacts are available might be an okay compromise?